config/haproxy.md

```diff
@@ -0,0 +1,23 @@
+<!-- [This file is part of the zabbix-agent-osso package] -->
+
+# Haproxy
+
+The haproxy script and template are a minimal monitoring solution for haproxy
+servers to monitor that all backends are availible, and the server is not
+running into its max connection (maxconn) setting.
+
+The haproxy loadbalancer script works by getting all sockets like
+'/var/run/haproxy/{CUSTOMER}/{INSTANCE}.stats'.
+
+The stats socket should be enabled in the haproxy configs like:
+```
+stats socket /var/run/haproxy/{CUSTOMER}/{INSTANCE}.stats level admin
+```
+The script requires python3
+
+With the goal of keeping the items to a minimum, the zabbix template creates
+items for only a subset of data points in the `haproxy --get-all-data` output.
```

Comment:
I'd reword this to:

> To keep the zabbix items to a minimum, the zabbix template only creates items
for a subset of the data found in the `haproxy --get-all-data` command output.